wikipedia-dark.user.css

```diff
@@ -1,7 +1,7 @@
 /* ==UserStyle==
 @name         Wikipedia Dark
 @namespace    StylishThemes
-@version      3.4.0
+@version      3.4.1
```

Comment:
Please dont include any release commits with PR's,

Thanks.

Reply:
My mistake. I've updated it to match master to avoid any conflicts.

Reply:
Dont edit this value at all please, master changes and this will screw it up.